config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -2,6 +2,51 @@
 # see genjobs.go, which you can run with hack/update-config.sh
 presubmits:
   kubernetes-security/kubernetes:
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    cluster: security
+    context: pull-security-kubernetes-e2e-aws-eks-1-11-prod-conformance
```

Comment:
we don't normally run conformance in presubmit - why do we have a presubmit for this at all?

Reply:
can we trust how this job is built to not leak any info? otherwise we should update the code that generates the security repo jobs to exclude it. The other jobs only upload to restricted locations because embargoed CVE fixes are tested here.

Reply:
> we don't normally run conformance in presubmit - why do we have a presubmit for this at all?

It's just for debugging CI (explained below). I think those can be any bunch of e2e tests for our current purpose.

> can we trust how this job is built to not leak any info?

Could you expand on this? What should be the criteria for 'trusted'? Also if it's not mandatory to define such a security job corresponding to the normal one, we can get rid of it FMU.

Reply:
trusted -> known to keep the outputs isolated. EG for the GCE jobs the outputs are uploaded to a private bucket, as are logs, and the clusters happen in an isolated project. things are similarish for the kops one.

Reply:
I believe we already drop some jobs outright when generating these, it's not clear if we should here.

Reply:
@shyamjvs @BenTheElder 

> for the GCE jobs the outputs are uploaded to a private bucket

We only upload kubeconfig to our test account S3 bucket. We do not upload logs, and just let upstream kubetest dump logger handle all artifacts.

And here's the code for kubeconfig upload:

https://github.com/aws/aws-k8s-tester/blob/7ce5b96f09df517aee6ad48dbc2b5ed1bae910f4/internal/eks/s3/plugin_embedded.go#L147-L155

It's only uploaded to a private bucket.

Please let me know if there's any other requirement.

Hopefully, we figure this out and remove this from presubmits by this or next week.

Reply:
thanks, that SGTM.

also to be clear: I am totally erring on the side of caution here. When the subpath CVE came through these jobs were in especially bad shape and we had to shut things down before we could spin them up... trying to avoid that going forward :-)